src/shapes/itext.class.ts

```diff
@@ -170,6 +168,7 @@ export class IText extends ITextClickBehaviorMixin<ITextEvents> {
    */
   _set(key: string, value: any) {
     if (this.isEditing && this._savedProps && key in this._savedProps) {
+      // @ts-expect-error irritating TS
       this._savedProps[key] = value;
```

Comment:
to add context around this, i think is because we mutate more state other than `isEditing` when going editing mode.
I ll open a task to fix it seems it seems just wrong as a pattern.

Reply:
I agree